arm-sql/2015-05-01-preview/swagger/encryptionProtectors.json

```diff
@@ -0,0 +1,355 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/encryptionProtector": {
+      "get": {
+        "tags": [
+          "EncryptionProtectors"
+        ],
+        "description": "Returns a list of the server encryption protectors",
+        "operationId": "Servers_ListEncryptionProtectors",
```

Comment:
EncryptionProtector is a singleton, so this is following the pattern for naming singleton operations. See [blob auditing policies](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-sql/2015-05-01-preview/swagger/blobAuditingPolicies.json) for example of the pattern.